packages/appstore/src/components/add-more-wallets/add-more-wallets.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Text, Loading } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { useAvailableWallets } from '@deriv/hooks';
+import { observer } from '@deriv/stores';
+import CarouselContainer from './carousel-container';
+import AddWalletCard from './wallet-add-card';
+
+import './add-more-wallets.scss';
+
+const AddMoreWallets = observer(() => {
+    const { data, isLoading } = useAvailableWallets();
+
+    if (!data) return <Loading is_fullscreen={false} />;
+
+    return (
+        <div className='add-wallets' data-testid='dt-add-wallets'>
```

Comment:
```suggestion
        <div className='add-wallets'>
```
if we are going to getByText as ameerul mentioned above